readme: release announcement (#1142)
simplex-chat/simplex-chat: blog/20220928-simplex-chat-v4-encrypted-database.md

### For developers - TypeScript SDK for integrating with SimpleX Chat

> It's been quite some time since it's been possible to create a chat-bot using SimpleX Chat as a library - but you would have to either write Haskell code, and this is not the most widely used language, or use foreign functions interface in any other language.

Reviewer: ```suggestion
It's been quite some time since it's been possible to create a chat-bot using SimpleX Chat as a library - but you had to either write Haskell code or use foreign functions interface in any other language.
```
